### ISSUE TITLE
ChefSpec

### ISSUE THREAD

**Issue by rmoriz** [logentries/le_chef#51]
Hi,

when having this cookbook in a run_list, ChefSpec fails:

```
================================================================================
Recipe Compile Error in /var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/base/recipes/default.rb
================================================================================

Errno::ENOENT
-------------
No such file or directory - apt-cache

Cookbook Trace:
---------------
  /var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/logentries_agent/recipes/install.rb:60:in `from_file'
  /var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/logentries_agent/recipes/default.rb:19:in `from_file'
  /var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/base/recipes/_logentries.rb:13:in `from_file'
  /var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/base/recipes/default.rb:36:in `from_file'

Relevant File Content:
----------------------
/var/folders/48/v3jpvqg155d2qfktth99_8t40000gp/T/d20160212-58372-85rhz7/cookbooks/logentries_agent/recipes/install.rb:

 53:  deamon_package_resource = package 'logentries-daemon' do
 54:    notifies :delete, "file[#{dont_run_file}]", :immediately
 55:  end
 56:  
 57:  # if logentries-daemon package is not already installed during compile phase
 58:  # of this chef run, we want to create the init script's "dont_run" file long
 59:  # enough to install it (to prevent it from auto-starting with no config)
 60>> if deamon_package_resource.provider_for_action(:install).load_current_resource.version.nil?
 61:    resources("file[#{dont_run_file}]").action(:create)
 62:  end
 63:  

```

**Comment by jheung-r7** [comment]
hi @rmoriz, just updating to say I've been able to replicate the issue and working on a fix at the moment. Sorry for the delay in getting back to you on this

**Comment by rmoriz** [comment]
I've made a workaround at https://github.com/pspdfkit-ops/le_chef/commit/389a8c33e722d96da201386bb6aabeb71db74d2e

**Comment by jheung-r7** [comment]
Great, let me have a look, I'll add you on the PR

**Comment by udangel-r7** [comment]
Should be closed with #52
